types/mui-datatables/index.d.ts

```diff
@@ -12,38 +12,37 @@ import * as React from 'react';
 export type Display = 'true' | 'false' | 'excluded';
 export type SortDirection = 'asc' | 'desc';
 export type FilterType = 'dropdown' | 'checkbox' | 'multiselect' | 'textField' | 'custom';
-export type Responsive = 'stacked' | 'scroll';
+export type Responsive = 'stacked' | 'scrollMaxHeight' | 'scrollFullHeight';
 export type SelectableRows = 'multiple' | 'single' | 'none';
 
 interface MUIDataTableData {
-    index: number;
     data: Array<object | number[] | string[]>;
+    index: number;
 }
 
 interface MUIDataTableStateRows {
     data: string[];
     lookup: any;
 }
 
-export interface MUIDataTableState {
-    announceText: string | null;
+interface MUIDataTableState {
```

Comment:
Why is this no longer exported?

Reply:
apologies. Ill resolve this. this might have been left over from my last pull of this repo